contracts/SportMarkets/Rundown/TherundownConsumer.sol

```diff
@@ -473,7 +473,11 @@ contract TherundownConsumer is Initializable, ProxyOwned, ProxyPausable {
 
             uint index = sportsManager.numActiveMarkets() - 1;
             if (!isSportTwoPositionsSport(sportsIdPerGame[_gameId])) {
```

Comment:
we might have a sport that is more than two positional that is not soccer (no double chance).
Might be fine for now, but an alternative is to have a list of sportIDs that support double chance